supportedBackends/google/batch/src/main/scala/cromwell/backend/google/batch/actors/GcpBatchInitializationActor.scala

```diff
@@ -72,8 +72,8 @@ class GcpBatchInitializationActor(batchParams: GcpBatchInitializationActorParams
   private lazy val gcsCredentials: Future[Credentials] = gcpBatchConfiguration.batchAttributes.auths.gcs
     .retryCredentials(workflowOptions, List(StorageScopes.DEVSTORAGE_FULL_CONTROL))
 
-  // Credentials object for the Genomics API
-  private lazy val genomicsCredentials: Future[Credentials] = gcpBatchConfiguration.batchAttributes.auths.genomics
+  // Credentials object for the Batch API
+  private lazy val batchCredentials: Future[Credentials] = gcpBatchConfiguration.batchAttributes.auths.batch
     .retryCredentials(workflowOptions,
                       List(
                         CloudLifeSciencesScopes.CLOUD_PLATFORM,
```

Comment:
I have tried removing this (`CloudLifeSciencesScopes.CLOUD_PLATFORM `) and `GenomicsScopes.GENOMICS` but centaur tests failed.

Does anyone know whether we should use a different scope? I'd expect that we shouldn't need anything lifesciences/genomics related (see https://github.com/broadinstitute/cromwell/actions/runs/8815748415/job/24198334692).